README.dev.md

```diff
@@ -92,6 +95,17 @@ virtualenv -p python .venv
 .venv\Scripts\activate.ps1
 python -m pip install -r requirements.txt
 ```
+
+#### Conda virtual environment
+To setup RP2 in a conda environment, enter the following commands:
```

Comment:
Can you leave the reference to Powershell? I.e.:  "open a PowerShell window and enter the following commands"